src/commands/append/fact.ts

```diff
@@ -0,0 +1,46 @@
+import {TokenAwareCommand} from '../../lib/token-command'
+import {initClient} from '../../lib/client'
+import * as bodyInputHelper from '../../lib/body-input-helper'
+
+export default class AppendFact extends TokenAwareCommand {
+  static description = 'Appends a factual event to the ledger'
+
+  static examples = [
+    `$ evently append:fact <<EVENT
+{
+  entity: "thermostat",
+  event: "temperature-recorded",
+  key: "thermostat1",
+  meta: {causation: "14rew3494"},
+  data:{celsius: 18.5}
+}
```

Comment:
I imagine there are better examples for all of these!

Reply:
Sorry was not a critique, i just re-used the same thermostat example just with the different parameters. So if you got something that's more appropriate for each event-type (fact, serial, atomic, lmk!)